scripts/run_example.sh

```diff
@@ -1,3 +1,5 @@
+set -e
```

Comment:
Without this, the script just plows ahead in the face of errors, making things hard to debug